flipperdevices/flipperzero-firmware
# [FL-3271] cubewb: updated to v1.16.0

# What's new

- [FL-3271] Updated radio stack & LL libraries to v1.16.0
- Fixed advertising start without delay on BLE client disconnect leading to C2 lock-up

# Verification 

- Verify that C2 no longer locks up on repeated stat/exit from BLE Remote app
- Verify that C2 no longer locks up on RPC session end on desktop/battery info with logging disabled

# Checklist (For Reviewer)



[FL-3271]: https://flipperzero.atlassian.net/browse/FL-3271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
